pommerman/agents/http_agent.py

```diff
@@ -0,0 +1,78 @@
+'''The http agent.'''
```

Comment:
Can you add a brief note as to what people would use this for please?

Reply:
I would also recommend consistently capitalizing "HTTP" as it is a abbreviation.